e2e-tests/web/test/specs/navigateToDashboard.e2e.ts

```diff
@@ -0,0 +1,34 @@
+import LandingPage from '../pageobjects/landing.page';
```

Comment:
Why `test/specs` directory instead of just `test` or `specs`?

Reply:
This is the autogenerated structure. But I think the main reason ist to create a container for both pageobjects and specs